chapter3/astroml_chapter3_Overview_of_Probability_and_Random_Variables.ipynb

```diff
@@ -0,0 +1,845 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "71655f70",
+   "metadata": {},
+   "source": [
+    "# Overview of Probability and Random Variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1b8b4c8",
+   "metadata": {},
+   "source": [
+    "This function adjusts `matplotlib` settings for a uniform feel in the textbook. Note that with `usetex=True,` fonts are rendered with LaTeX. This may result in an error if LaTeX is not installed on your system. In that case, you can set `usetex` to `False.`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "4c48b283",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if \"setup_text_plots\" not in globals():\n",
+    "    from astroML.plotting import setup_text_plots\n",
+    "setup_text_plots(fontsize=8, usetex=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2823b358",
+   "metadata": {},
+   "source": [
+    "## Probability axioms"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f224fef1",
+   "metadata": {},
+   "source": [
+    "Given an event $A$, such as the outcome of a coin toss, we assign it a real number $p(A)$, called the probability of $A$. Note that $p(A)$ could also correspond to a probability that a value of $x$ falls in a $dx$ wide interval around $x$. To qualify as a probability, $p(A)$ must satisfy three Kolmogorov axioms:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9179164d",
+   "metadata": {},
+   "source": [
+    "1. $p(A) \\geq 0$ for each $A$.\n",
+    "2. $p(\\Omega) = 1$, where $\\Omega$ is the set of all possible outcomes.\n",
+    "3. If $A_1$, $A_2$, . . . are disjoint events, then $p (\\bigcup^{\\infty}_{i=1}A_i) = \\sum_{i=1}^{\\infty}p(A_i)$ where $\\bigcup$ stands for “union.”\n",
+    "\n",
+    "Several useful rules can be derived as a consequence of these axioms."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c067a63",
+   "metadata": {},
+   "source": [
+    "**Sum rule**:  The probability that the union of two events, $A$ and $B$, will happen is given by,\n",
+    "\n",
+    "$$\\qquad\\qquad p(A \\cup B) = p(A) + p(B) - p(A \\cap B)\\qquad\\qquad\\qquad(1)$$\n",
```

Comment:
I wonder whether we could make the eq numbering work out of the box, rather than hard-wiring it ourselves. I'm opening a follow-up issue for it (it certainly points beyond the scope of this PR).

https://github.com/astroML/astroML-notebooks/issues/62